ChatApp/models.py

```diff
@@ -87,4 +87,60 @@ def translationLanguage(id):
           print('Exception3が発生しています')
           abort(500)
       finally:
-          cur.close()
\ No newline at end of file
+          cur.close()
+
+  def getChannelAll():
+    try:
+        conn = DB.getConnection()
+        cur = conn.cursor()
+        sql = "SELECT * FROM CHANNELS;"
+        cur.execute(sql)
+        channels = cur.fetchall()
+        return channels
+    except Exception as e:
+        print('エラー:' + e)
+        abort(500)
+    finally:
+        cur.close()         
+  
+  def getChannelById(channel_id):
+    try:
+        conn = DB.getConnection()
+        cur = conn.cursor()
+        sql = "SELECT * FROM CHANNELS WHERE channel_id=%S;"
+        cur.execute(sql, (channel_id))
+        channel = cur.fetchone()
+        return channel
+    except Exception as e:
+        print('エラー:' + e)
```

Comment:
ここ自分もミスってました！
動確の時にエラーになったので、自分のとこも含めて修正して頂ければと思います！
すみません ><
```suggestion
    except Exception as e:
        print(f'エラー: {e}')
```